globals.go

```diff
@@ -604,7 +604,7 @@ var (
 )
 
 const (
-	DefaultServiceConfigurationFileName = "/etc/mci_settings.yml"
+	DefaultServiceConfigurationFileName = "/etc/mci_settings.yml" // Unused
```

Comment:
Should I delete these unused variables in this PR? I stumbled upon them while working on this